package.json

```diff
@@ -7,8 +7,10 @@
     "develop": "gulp develop",
     "build-dist-js": "gulp build-dist-js",
     "test": "gulp test-coverage",
-    "lint": "eslint src example --ext .jsx --ext .js"
-},
+    "lint": "eslint src example --ext .jsx --ext .js",
+    "prepublish": "gulp build-dist-scss"
+  },
+  "style": "./dist/css/react-calendar.css",
```

Comment:
:question: What tooling would this affect? ie if you use Webpack, will this automagically be included now?
Will this affect existing projects? ie would we need to make this change as part of a major release?